File: test/fixtures/clojure/redirect_variable_data.clj
(require '[clj-http.client :as client])

(client/post "http://localhost:28139" {:headers {"Content-Type" "application/x-www-form-urlencoded"}
                                       :body (str "foo&" (clojure.java.io/file (str (System/getenv "FILENAME"))))})

File: test/fixtures/r/redirect_variable_data.r
library(httr)

headers = c(
  `Content-Type` = "application/x-www-form-urlencoded"
)

data = paste0("foo&@", Sys.getenv("FILENAME"))

res <- httr::POST(url = "http://localhost:28139", httr::add_headers(.headers=headers), body = data)
